k8s/pkg/apis/longhorn/v1beta2/recurringjob.go

```diff
@@ -2,14 +2,15 @@ package v1beta2
 
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-// +kubebuilder:validation:Enum=snapshot;snapshot-cleanup;snapshot-delete;backup
+// +kubebuilder:validation:Enum=snapshot;snapshot-force-create;snapshot-cleanup;snapshot-delete;backup
 type RecurringJobType string
 
 const (
-	RecurringJobTypeSnapshot        = RecurringJobType("snapshot")         // periodically create snapshots
-	RecurringJobTypeSnapshotCleanup = RecurringJobType("snapshot-cleanup") // periodically purge removable snapshots and system snapshots
-	RecurringJobTypeSnapshotDelete  = RecurringJobType("snapshot-delete")  // periodically remove and purge all kinds of snapshots that exceed the retention count
-	RecurringJobTypeBackup          = RecurringJobType("backup")           // periodically create snapshots then do backups
+	RecurringJobTypeSnapshot            = RecurringJobType("snapshot")              // periodically create snapshots
```

Comment:
Add one more statement explaining the new behavior

Reply:
Added. Thanks